### PR TITLE
Update experience commenced labels to "Years of experience".

### LINKED ISSUE #5205: Replace " Experience commenced on " field with "No. of Years of experience" in user creation page
**Describe the bug**

The " experience commenced on " field in the user creation form is confusing the users while creating a new doctor id.

**To Reproduce**

1. Go to https://care.coronasafe.in/users/add
2. See error

**Expected behavior**

- [ ] Replace " Experience commenced on " field with "No. of Years of experience" in user creation page
- [ ] it should be a mandatory field.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #5205

Rename label "Experience Commenced On" to "Years of experience" in UserAdd and UserProfile components. This change provides a more clear and concise label that describes the input field. The suffix for the field has also been updated to reflect the new label. This change improves the user experience by making the input field more intuitive and easier to understand.

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

**Review by rithviknishad — APPROVED** [review]
I thought since the label has changed to `Years of Exp.` we had to change the field also to a number input and calculate experience_commenced_on based on input before firing the request

**Comment by Ashesh3** [comment]
> I thought since the label has changed to `Years of Exp.` we had to change the field also to a number input and calculate experience_commenced_on based on input before firing the request

I thought the same, @nihal467 could you take a look if we need to convert the field to a number input?

**Comment by nihal467** [comment]
@Ashesh3 @rithviknishad  we need to convert the field, which will be a simple input field just to input the number of experience in years

**Comment by Ashesh3** [comment]
> @Ashesh3 @rithviknishad we need to convert the field, which will be a simple input field just to input the number of experience in years

Done!

image

image

File: src/Components/Users/UserAdd.tsx [hunk 1]
```diff
@@ -684,15 +683,12 @@ export const UserAdd = (props: UserProps) => {
                     placeholder="Qualification of the Doctor"
                   />
 
-                  <MonthFormField
+                  <TextFormField
                     {...field("doctor_experience_commenced_on")}
                     required
```
Comment (rithviknishad) [inline src/Components/Users/UserAdd.tsx:688]: 👀 
image

```suggestion
                    required
                    min={0}
```

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Users/UserAdd.tsx (hunk 1 above)
Comment (Ashesh3) [reply src/Components/Users/UserAdd.tsx:688]: 😬

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@Ashesh3 could you also change the field to numeric input in the profile update page?

(reported by @nihal467)

**Comment by Ashesh3** [comment]
> @Ashesh3 could you also change the field to numeric input in the profile update page?
> 
> (reported by @nihal467)

Done!

image

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**